### PR TITLE
Inherits `feat: Worker::patch` PR

### PR DESCRIPTION
**Pull request by shariffdev**
Closes https://github.com/near/workspaces-rs/issues/250
This PR takes in work done by @ChaoticTempest and tries to address code reviews and finalise this PR https://github.com/near/workspaces-rs/pull/264 which has become stale.

### CODE REVIEW

**Comment by shariffdev** [comment]
@ChaoticTempest @frol  I have picked this up issue and continued from where @ChaoticTempest left off (Hope that's okay)

File: workspaces/src/rpc/patch.rs [hunk 1]
```diff
@@ -107,68 +109,232 @@ impl<'a> ImportContractTransaction<'a> {
...
+    /// Patch and overwrite the info contained inside an [`Account`] in sandbox.
+    pub fn account(mut self, account: AccountDetailsPatch) -> Self {
+        self.account_updates.push(AccountUpdate::Update(account));
+        self
+    }
+
+    /// Patch and overwrite the info contained inside an [`Account`] in sandbox. This
+    /// will allow us to fetch the current details on the chain and allow us to update
+    /// the account details w.r.t to them.
+    pub fn account_from_current<F: 'static>(mut self, f: F) -> Self
```
Comment (frol) [inline workspaces/src/rpc/patch.rs:186]: Please, add test for this method.

**Review by frol — APPROVED** [review]
Overall, looks good to me. Ready to merge it once the other comment is addressed.

File: workspaces/src/rpc/patch.rs (hunk 1 above)
Comment (shariffdev) [reply workspaces/src/rpc/patch.rs:186]: Done

**Pull request merged**